content/number-theory/euclid.h

```diff
@@ -2,14 +2,12 @@
  * Author: Unknown
  * Date: 2002-09-15
  * Source: predates tinyKACTL
- * Description: Finds the Greatest Common Divisor to the integers $a$ and $b$.
- * Euclid also finds two integers $x$ and $y$, such that $ax+by=\gcd(a,b)$.
+ * Description: Finds two integers $x$ and $y$, such that $ax+by=\gcd(a,b)$. If
+ * you just need gcd, use the built in __gcd instead
```

Comment:
```suggestion
 * you just need gcd, use the built in \texttt{\_\_gcd} instead.
```
(just guessing at the underscore, but I think latex will parse them as math otherwise?)